app/assets/stylesheets/local/custom.scss

```diff
@@ -18,6 +18,6 @@ form.search {
   }
 }
 
-p.help-text {
+p.help-text, #superseded_style  {
```

Comment:
This would be best to be a class instead of an ID.

It also helps to keep any app-specific classes prefixed with `app`, for instance:
`p.app-help-text, app-superseded-style { ... }`